docs/overview/index.md

```diff
@@ -1,17 +1,105 @@
+
+The California DMV is also using Tezos for its project to [put car titles on the blockchain](https://fortune.com/crypto/2023/01/26/california-announces-dmv-run-blockchain-through-partnership-with-tezos/).
+
+[Sword Group](https://www.sword-group.com/2020/09/28/sword-launches-tezos-digisign/) an international technology company, launched DigiSign, an open-source tool built on Tezos that enables users to digitally sign, certify, and verify the authenticity of digital documents.
+
+### Tezos in gaming
+
+Recently, the [Tezos Unity SDK](../unity) promises to make blockchain game development easier and faster. It allows for the addition of web3 features such as allowing players to link their accounts across games, mint, and trade in-game items and currencies, and show off their ranks and accomplishments on public, on-chain leaderboards.
+
+## What makes Tezos different?
+
+Here are some of the features that make Tezos different from other blockchains:
+
+### Tezos can upgrade itself
+
+Tezos has a built-in capability to upgrade itself, which allows the network to evolve without requiring a hard fork. Anyone can propose an upgrade to the protocol and have it adopted by the network without compromising the platform's stability or causing fragmentation. This feature allows Tezos to adapt regularly to new technologies and to address user needs rapidly.
+
+### Everyone can participate in governance
+
+Anyone who holds XTZ — the chain's native token — can propose changes to how Tezos works, such as changes to gas fees and block times, new features such as Smart Rollups, or even major changes like how the consensus mechanism works.
+
+### Formal verification ensures trust and code quality
+
+*Formal verification* is a process that ensures that a smart contract does what it says it does and has no side effects. Formal verification reduces errors, bugs, and security vulnerabilities in contracts and allows users to trust them. For more information, see [Formal Verification](https://opentezos.com/formal-verification) on opentezos.com.
+
+### Tezos uses proof of stake
+
+The proof-of-stake consensus model eliminates the need for high energy use, making it the "green" choice for blockchains. Instead of competing to achieve consensus as in proof-of-work models, Tezos nodes (called *bakers*) stake Tezos tokens to earn the right to create blocks and receive rewards. Users who want to participate without running a node themselves can delegate tokens to a baker for a share of the rewards. The bakers keep control of their tokens in that they can unstake them later; and delegators keep complete control of their tokens. Tezos's approach to consensus has been described as [Liquid Proof of Stake](https://medium.com/tezos/liquid-proof-of-stake-aec2f7ef1da7).
```

Comment:
update for staking?
"can delegate tokens to a baker" -> "can delegate tokens to a baker or stake tokens on behalf of a baker"

Reply:
also "The bakers keep control of their tokens" -> "The bakers and stakers keep control of their tokens"

Reply:
Changed to "Users who want to participate without running a node themselves can delegate tokens to a baker or stake tokens on behalf of a baker for a share of the rewards. The bakers and stakers keep control of their tokens in that they can unstake them later; delegators keep complete control of their tokens."